src/modules/pf3dTracker/src/CMakeLists.txt

```diff
@@ -2,14 +2,17 @@
 # Authors: Lorenzo Natale, Matteo Taiana, Ugo Pattacini
 # CopyPolicy: Released under the terms of the GNU GPL v2.0.
 
+cmake_minimum_required(VERSION 2.6)
+set(PROJECTNAME pf3dTracker)
```

Comment:
Please change the CMakeLists structure to only have one at the root of the folder (pf3dTracker).  I have made the following changes for you. 

``` cmake
# Copyright: (C) 2010 RobotCub Consortium
# Authors: Lorenzo Natale, Matteo Taiana, Ugo Pattacini
# CopyPolicy: Released under the terms of the GNU GPL v2.0.

cmake_minimum_required(VERSION 2.6)
project(pf3dTracker)

file(GLOB folder_source src/*.cpp)
file(GLOB folder_header include/iCub/*.hpp)

source_group("Source Files" FILES ${folder_source})
source_group("Header Files" FILES ${folder_header})

include_directories(${PROJECT_SOURCE_DIR}/include ${OpenCV_INCLUDE_DIRS} ${GSL_INCLUDE_DIRS} ${YARP_INCLUDE_DIRS})

add_executable(${PROJECT_NAME} ${folder_header} ${folder_source})
target_link_libraries(${PROJECT_NAME} ${OpenCV_LIBRARIES} ${YARP_LIBRARIES} ${GSL_LIBRARIES})
install(TARGETS ${PROJECT_NAME} DESTINATION bin)
```